packages/atomic/src/components/search/atomic-generated-answer/source-citations.tsx

```diff
@@ -0,0 +1,38 @@
+import {GeneratedAnswerCitation} from '@coveo/headless';
+import {FunctionalComponent, h} from '@stencil/core';
+
+interface SourceCitationsProps {
+  label: string;
+  citations: GeneratedAnswerCitation[];
+  onCitationClick: (citation: GeneratedAnswerCitation) => void;
+}
+
+export const SourceCitations: FunctionalComponent<SourceCitationsProps> = (
```

Comment:
I recommend copying a lot from `atomic-smart-snippet-source`, especially the analytics. May need to  do some of that in a separate PR.

Reply:
Would you mind if we improve the analytics binding later on? I think this would make us miss our deadline given the only possible time for a release is tomorrow.

Reply:
That's fine